src/openforms/emails/templates/emails/admin_digest.html

```diff
@@ -77,3 +77,53 @@ <h5>{% trans "Configuration problems" %}</h5>
         {% endfor %}
     </ul>
 {% endif %}
+
+{% if invalid_certificates %}
+    <h5>{% trans "Invalid certificates" %}</h5>
+    <ul>
+        {% for invalid_certificate in invalid_certificates %}
+            <li>
+                <p>
+                    <!-- Expiring certificates -->
+                    {% if invalid_certificate.expiry_date %}
+                        {% blocktranslate with certificate_label=invalid_certificate.label expiry_date=invalid_certificate.expiry_date trimmed %}
+                            '{{ certificate_label }}' certificate is about to expire on {{ expiry_date }}
+                        {% endblocktranslate %}
+                    {% else %}
+
+                        <!-- No private key provided -->
+                        {% if invalid_certificate.is_valid_pair == None %}
+                            {% blocktranslate with certificate_label=invalid_certificate.label trimmed %}
+                                No private key for certificate '{{ certificate_label }}' was provided.
+                            {% endblocktranslate %}
+                        {% endif %}
```

Comment:
`is_valid_pair` `None` means that nothing can be said about validity. If the certificate is of type `cert_only`, then this is fine because no private key is expected.

You can only say something meaningful about `is_valid_pair == False` because that means something is actually wrong.